### PR TITLE
fix(dropzone.stories.tsx): Preview updates correctly & deleted file is removed

### PR DESCRIPTION
**Pull request by silvalaura**
Fix https://github.com/cengage/react-magma/issues/570 & https://github.com/cengage/react-magma/issues/571

1. Updating the Dropzone preview for Image/Text/CSV when controls are toggled
2. Removing the image or file preview after they have been deleted

_lots of spacing changes because I ran prettier on the file_

### CODE REVIEW

**Review by ccedrone — APPROVED** [review]

**Review by ccedrone — APPROVED** [review]

File: packages/dropzone/src/components/dropzone/Dropzone.stories.tsx [hunk 1]
```diff
@@ -1,45 +1,51 @@
 import { useState } from 'react';
 import { Dropzone, DropzoneProps, OnSendFileProps } from './Dropzone';
+import { Preview } from './Preview';
 import { Textarea, Datagrid } from 'react-magma-dom';
 
-function csvJSON(csv: string){
-  var lines=csv.split("\n");
```
Comment (greenervigil) [inline packages/dropzone/src/components/dropzone/Dropzone.stories.tsx:6]: I think we should be using `let`

**Review by greenervigil — CHANGES_REQUESTED** [review]
we should update to use `let` and `const`

File: .changeset/new-owls-perform.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"react-magma": patch
```
Comment (greenervigil) [inline .changeset/new-owls-perform.md]: this needs to change

**Review by greenervigil — CHANGES_REQUESTED** [review]
just need to update the changeset file package

File: .changeset/new-owls-perform.md (hunk 2 above)
Comment (silvalaura) [reply .changeset/new-owls-perform.md]: Can you refresh? It is updated: 
image

[d35e03f](https://github.com/cengage/react-magma/pull/615/commits/d35e03f99f550bbde561662e296686c54510cc5c)

File: .changeset/new-owls-perform.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+"react-magma-dom": patch
```
Comment (greenervigil) [inline .changeset/new-owls-perform.md:2]: this is in the dropzone package not react-magma-dom

**Review by greenervigil — CHANGES_REQUESTED** [review]
Wrong package in your changeset

File: .changeset/new-owls-perform.md (hunk 3 above)
Comment (silvalaura) [reply .changeset/new-owls-perform.md]: 🤦‍♀️  fixed

**Review by greenervigil — APPROVED** [review]

**Pull request merged**